src/server/object_services/object_server.js

```diff
@@ -770,6 +770,20 @@ async function read_node_mapping(req) {
     };
 }
 
+
+function _convert_rpc_params_to_bucket_policy_req(rpc_params) {
+
+    let req_query = null;
```

Comment:
Why do we init it to `null`?

Reply:
just for clarity. explicitly saying either we return version_id in query or it will be null..